wheel_matrix.yml

```diff
@@ -1,4 +1,44 @@
 packages:
+  bcrypt:
+    versions:
+      latest:
+        wheels:
+        - platform_tag: freebsd_12_2_release_amd64
+          platform_instance: freebsd/12.2
+          platform_arch: x86_64
+          python:
+          - tag: cp38
+            abi: abi3
```

Comment:
FreeBSD 12.2 is only used in stable-2.12 and stable-2.11 -- not devel. Normally we wouldn't add a non-current version here, but this is easier than manually running a build. We can remove it from the matrix once it's built.